pivotal-api-test/target/test-classes/features/project/UpdateProject.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Update project settings
```

Comment:
This feature file and DeleteProject.feature have the same precondition (Background)
Please join them.